indexer/services/roundtable/__tests__/tasks/update-compliance-data.test.ts

```diff
@@ -676,8 +676,7 @@ async function setupInitialSubaccounts(
   deltaSeconds: number,
 ): Promise<void> {
   const subaccounts: SubaccountFromDatabase[] = await SubaccountTable.findAll({
-    address: testConstants.defaultAddress,
-    subaccountNumber: testConstants.defaultSubaccount.subaccountNumber,
```

Comment:
Functionally, this is the same as before - since subaccount number was 0 and there was no filtering happening